linux_os/guide/services/ldap/openldap_client/package_openldap-clients_removed/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: rhel7,ol7,rhel8,ol8,fedora,rhv4,ocp4
```

Comment:
Nitpick: order the prodtypes alphabetically, this will reduce changes in the future.